nvflare/fuel/hci/server/file_transfer.py

```diff
@@ -308,3 +331,45 @@ def download_job(self, conn: Connection, args: List[str]):
     def info(self, conn: Connection, args: List[str]):
         conn.append_string("Server Upload Destination: {}".format(self.upload_dir))
         conn.append_string("Server Download Source: {}".format(self.download_dir))
+
+    def _validate_job(self, meta: dict, study_manager: StudyManagerSpec, fl_ctx: FLContext) -> Tuple[bool, str]:
+
+        study_name = meta.get(JobMetaKey.STUDY_NAME)
+        if not isinstance(study_name, str):
+            return False, "STUDY_NAME is expected in meta.json to be a str"
```

Comment:
should this be "Study Name" instead of "STUDAY_NAME" ?

Reply:
I think "study_name" is better since that's the field name in the JSON.